golang/cosmos/app/upgrade.go

```diff
@@ -157,6 +171,26 @@ func replacePriceFeedsCoreProposal(upgradeName string) (vm.CoreProposalStep, err
 	)
 }
 
+func terminateGovernorCoreProposal(upgradeName string) (vm.CoreProposalStep, error) {
+	// targets is a slice of "$boardID:$instanceKitLabel" strings.
+	var targets []string
+	switch getVariantFromUpgradeName(upgradeName) {
+		case "MAINNET":
+			targets = []string{"board052184:stkATOM-USD_price_feed"}
```

Comment:
Yeah we usually go for the "variant" param being plumbed into the JS proposal builder. But I similarly don't care enough one way or the other, since it's simple enough here.

Reply:
This allows reuse of the builder script without modification for post-upgrade core-eval proposals to terminate _other_ instances without hard-coding potential targets.